IRremoteESP8266.h

```diff
@@ -232,18 +232,19 @@ class IRsend
   void sendKelvinator(unsigned char data[]);
   void sendSherwood(unsigned long data, int nbits=32, unsigned int repeat=1);
   void sendMitsubishiAC(unsigned char data[]);
-  void enableIROut(int khz);
+  void enableIROut(unsigned int khz, uint8_t duty=50);
```

Comment:
I see that this going to break a majority of programs/sketches out there. So, can we have the overloaded method so as to maintain compatibility with those programs. 
e.g. 
```
enableIROut(unsigned int khz) {
  uint8_t duty=50 //we set a default duty cycle
  IRSend.enableIROut(khz, duty);
}
```